<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="infoWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>220</width>
     <height>241</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="pidLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="pidLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>PID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vidLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="vidLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>VID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="manuLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="manuLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Manufa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prodLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="prodLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="seriLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="seriLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Serial</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="indxLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="indxLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Index</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="ctrlWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>220</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ejectButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Eject</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="readWidget" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QTextBrowser" name="readBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>300</width>
      <height>130</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="readlabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>3</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Read Data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="readButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="sendWidget" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>300</width>
     <height>200</height>
    </rect>
   </property>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>300</width>
      <height>130</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="sendLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>Send Data</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sendLineEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please enter hex data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="ledWidget" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <widget class="QCheckBox" name="led6CheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
font: 11pt &quot;Calibri&quot;;
}
QCheckBox::indicator:checked{
border:1px solid black;
border-radius:6px;
background-color: rgb(0, 170, 255);
}
QCheckBox::indicator:unchecked{
background-color: rgb(255, 255, 255);
border:1px solid black;
border-radius:6px;
}</string>
    </property>
    <property name="text">
     <string>LED6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led4CheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
font: 11pt &quot;Calibri&quot;;
}
QCheckBox::indicator:checked{
border:1px solid black;
border-radius:6px;
background-color: rgb(255, 170, 0);
}
QCheckBox::indicator:unchecked{
background-color: rgb(255, 255, 255);
border:1px solid black;
border-radius:6px;
}</string>
    </property>
    <property name="text">
     <string>LED4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led3CheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
font: 11pt &quot;Calibri&quot;;
}
QCheckBox::indicator:checked{
border:1px solid black;
border-radius:6px;
background-color: rgb(255, 0, 0);
}
QCheckBox::indicator:unchecked{
background-color: rgb(255, 255, 255);
border:1px solid black;
border-radius:6px;
}</string>
    </property>
    <property name="text">
     <string>LED3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led5CheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
font: 11pt &quot;Calibri&quot;;
}
QCheckBox::indicator:checked{
border:1px solid black;
border-radius:6px;
background-color: rgb(170, 255, 0);
}
QCheckBox::indicator:unchecked{
background-color: rgb(255, 255, 255);
border:1px solid black;
border-radius:6px;
}</string>
    </property>
    <property name="text">
     <string>LED5</string>
    </property>
   </widget>
   <widget class="QLabel" name="ledLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Calibri&quot;;</string>
    </property>
    <property name="text">
     <string>LED</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ledAllCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
font: 11pt &quot;Calibri&quot;;
}
QCheckBox::indicator:checked{
border:1px solid black;
border-radius:6px;
	background-color: rgb(50, 50, 50);
}
QCheckBox::indicator:unchecked{
background-color: rgb(255, 255, 255);
border:1px solid black;
border-radius:6px;
}</string>
    </property>
    <property name="text">
     <string>LED ALL</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>170</y>
     <width>111</width>
     <height>251</height>
    </rect>
   </property>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>22</width>
      <height>200</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>BlankCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>650</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
